web/src/main/java/com/futurewei/alcor/web/entity/port/PortEntity.java

```diff
@@ -108,6 +109,7 @@
     @JsonProperty("revision_number")
     private int revisionNumber;
 
+    @JsonIgnore
```

Comment:
Issue #309 describes the issue clearly. Although this is an optional field, if we've known the right response, let us just fix the issue. Using JsonIgnore is sub-optimal.

Reply:
fixed in 7e07ce4